__tests__/intergration.test.js

```diff
@@ -145,6 +145,62 @@ describe("GET: /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("POST: /api/articles/:article_id/comments", () => {
+  test("200: successfully adds new comment to article and returns the posted comment", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "A brand new comment!",
+    };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body: { newComment } }) => {
+        expect(newComment).toHaveLength(1);
+        expect(newComment).toMatchObject(newComment);
+      });
+  });
+  test("400: sends an appropriate message and status when passed a malformed body", () => {
+    const malformedBody = {
+      username: "butter_bridge",
+      msg: 5,
+    };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(malformedBody)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("400: sends an appropriate message and status when passed a non-valid user", () => {
```

Comment:
Hmm, I would consider this a 404. This is because the username is valid in the sense of being the correct data type, but that it doesn't exist